Announcing Servo GitHub releases (#260)
servo/servo.org: _posts/2025-10-20-servo-0.0.1-release.md

---
layout:     post
tags:       blog
title:      "Servo 0.0.1 Release"
date:       2025-10-20

> summary:    This is a brief blog post explaining the goals and plans behind the new Servo releases on GitHub.

Reviewer: ```suggestion
summary:    A brief update on the goals and plans behind the new Servo releases on GitHub.
```

categories:
---

> Today Servo has released a version tagged [v0.0.1 - October 2025](https://github.com/servo/servo/releases/tag/v0.0.1), this is basically the same thing we were already providing for the [download page](https://servo.org/download) (note that now we also include arm64 mac binaries), but we are now starting to tag them as releases for future reference.

Reviewer: ```suggestion
Today, the Servo team has released new versions of the `servoshell` binaries for all our supported platforms, tagged [v0.0.1](https://github.com/servo/servo/releases/tag/v0.0.1). These binaries are essentially the same nightly builds that were already available from the [download page](https://servo.org/download), but now we have tagged them explicitly as releases for future reference.
```

The plan is to do releases like this on a monthly basis. The release process will use a recent nightly commit to do some additional manual testing in order to check there are no major issues with that version.

> So far we don't have plans to publish these releases on [crates.io](https://crates.io/), the idea is to simply make [tagged releases on GitHub](https://github.com/servo/servo/releases).

Reviewer: ```suggestion
So far we don't have plans to publish these releases on [crates.io](https://crates.io/) or platform-specific app stores, the idea is to simply make [tagged releases on GitHub](https://github.com/servo/servo/releases).
```
